colossalai/nn/layer/moe/layers.py

```diff
@@ -216,6 +216,23 @@ def forward(self, inputs: torch.Tensor, use_kernel: bool = False, ep_group: Opti
             return cb_weight, sec_mask
 
 
+class LinearGate(nn.Linear):

```

Comment:
@1SAA change the name to FP32LinearGater

Reply:
done